src/python/pants/backend/java/test/junit.py

```diff
@@ -5,6 +5,7 @@
 from dataclasses import dataclass
```

Comment:
Oh, hm. This file should probably be renamed to `backend/java/goals/junit_test_runner.py`. We don't need a dedicated `test/` folder.

Will you be touching this code in the near future? If so, fine for a followup.

Reply:
Unlikely I'll be touching this soon

Reply:
Okay, I think this would be good fix here, then.

Reply:
ack

Reply:
@Eric-Arellano I understand the package layout a bit better now: It looks like we've split the `java` package into `test` and `compile`, rather than having a single `goals` subpackage. If I'm going to make the change you're suggesting, I think it's probably something I should do holistically, rather than addressing just one file

Reply:
Okay, that's fine. Might be worth a dedicated PR like https://github.com/pantsbuild/pants/pull/12938.